pkg/virt-handler/vm.go

```diff
@@ -475,7 +475,8 @@ func (d *VirtualMachineController) hasTargetDetectedDomain(vmi *v1.VirtualMachin
 	return false, timeLeft
 }
 
-func (d *VirtualMachineController) clearPodNetworkPhase1(vmi *v1.VirtualMachineInstance) {
+// teardownNetwork performs network cache cleanup for a specific VMI.
```

Comment:
I'd like this to evolve over time (interface unplug). I can adjust the comment when the moment comes - just an heads up.